spec/requests/api/v1/events/events_spec.rb

```diff
@@ -99,11 +95,8 @@
     event.reload
     expect(event.name).to be
     expect(response_json).to eq(
-      'message' => 'Validation Failed',
-      'errors' => [
-        "Name can't be blank"
-      ]
+      'errors' => "Validation failed: Name can't be blank"
```

Comment:
So, reload the event, make sure the event.name isn't empty (is not blank) ... but an error says that can't be blank (is blank)?

Confused that both the `be` presence and the validation would succeed.

Also, line 77 ahead (not in this snippet) has a .reload on the event that isn't used.

The act of `reload` will do a DB call that isn't used and we always hope to minimize db call when possible -- especially in specs/tests.